torchvision/prototype/features/_label.py

```diff
@@ -1,57 +1,60 @@
 from __future__ import annotations
 
-from typing import Any, Optional, Sequence, Union
+from typing import Any, Optional, Sequence, Type, TypeVar, Union
 
 import torch
 from torch.utils._pytree import tree_map
 
 from ._feature import _Feature
 
 
-class Label(_Feature):
+F = TypeVar("F", bound="_Feature")
```

Comment:
Did you mean the following?
```suggestion
F = TypeVar("F", bound="_LabelBase")
```

If yes should it be `L` instead of `F`?